apps/accelerate/chatllama/chatllama/rlhf/actor.py

```diff
@@ -7,10 +7,11 @@
 from beartype.typing import Optional, Tuple
 from einops import rearrange
 from torch.utils.data import Dataset, DataLoader
-from config import ConfigActor
-from utils import TrainingStats
 
-from chatllama.llama_model import load_model
+from rlhf.config import ConfigActor
+from rlhf.utils import TrainingStats
+
+from llama_model import load_model
```

Comment:
please use absolute imports.
```suggestion
from chatllama.llama_model import load_model
from chatllama.rlhf.config import ConfigActor
from chatllama.rlhf.utils import TrainingStats
```